recipes-kernel/linux/linux-yocto_6.6.bbappend

```diff
@@ -0,0 +1,40 @@
+FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}-6.6:${THISDIR}/${PN}:"
```

Comment:
Add the `:tegra` override here and that should take care of the issue being reported by `yocto-check-layer`.